tests/testthat/test-correctionsataglance.R

```diff
@@ -70,9 +75,9 @@ test_that("calcStartSeq properly calculates the sequence of month start dates",
   expect_equal(length(startSeq4), 3)
 
   expect_equal(startSeq1, startDate1)
-  expect_equal(as.numeric(startSeq2), unlist(list(startDate1, monthDate2+startTime1, monthDate3+startTime1)))
+  expect_equal(as.numeric(startSeq2), unname(unlist(list(startDate1, monthDate2+startTime1, monthDate3+startTime1))))
   expect_equal(startSeq3, startDate2)
-  expect_equal(startSeq4, c(startDate2, monthDate2+startTime2, monthDate3+startTime2))
+  expect_equal(unname(startSeq4), unname(c(startDate2, monthDate2+startTime2, monthDate3+startTime2)))
```

Comment:
These needed to be unnamed in one of the earlier committed versions because it was returning named vectors with the input dates as the names. The `unname`s could go away now I think